app/LearnedLesson.php

```diff
@@ -6,6 +6,10 @@
 
 class LearnedLesson extends Model
```

Comment:
I think this is lesson object, there is provided ER diagram so please follow the Table name structure.

Reply:
-I'll change the name also.